module_cheatsheets/scRNA-seq_cheatsheet.md

```diff
@@ -0,0 +1,71 @@
+| Base `R`               | [`<-function(x) {     `code` }`](http://adv-r.had.co.nz/Functions.html)            | Create a function               | Creates a function that would take the defined parameters as input and execute the commands within the curly braces  |
+
+<div style="page-break-after: always;"></div>
+
+### `ggplot2`
+Read the `ggplot2` package documentation [**here**](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0). <br>
+A vignette on the usage of the `ggplot2` package can be found [**here**](https://cran.r-project.org/web/packages/ggplot2/vignettes/ggplot2-specs.html).
+
+| Library/Package      | Piece of Code                           | What it's called                     | What it does                                                                                                           |
+|----------------------|------------------------------------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|  
+| `ggplot2`            | [`qplot`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/qplot)                                 | Quick Plot               | Allows for the quick production of different types of plots when used in conjuction with layers created by other `ggplot2` functions   |
+| `ggplot2`            | [`geom_density()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_density)                 | Geom Density        | Displays a smoothed version of the `ggplot` or `qplot` based on the computed density estimate                                          |
+| `ggplot2`            | [`geom_vline()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_abline)                    | Vertical Line   | Adds a vertical reference line to a `ggplot` or  `qplot`                                                                               |
+| `ggplot2`            | [`theme_classic()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggtheme)                     | Classic Theme   | Displays `ggplot` or `qplot` without gridlines                                                                                         |
+| `ggplot2`            | [`facet_wrap()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/facet_wrap)                     | Facet Wrap      | Plots individual graphs using specified variables to subset the data                                                                   |
+| `ggplot2`            | [`ggtitle()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)                              | GG Title        | Sets the title of a `ggplot`                         |                                                       
+
+<div style="page-break-after: always;"></div>
+
+### `scran`
```

Comment:
I think you can combine the `scran` and `scater` headers + tables.

Reply:
Having now looked what is below, I would probably include `SingleCellExperiment` in this header and table. I would also link to https://osca.bioconductor.org/ in this section. You may find [this section](https://osca.bioconductor.org/data-infrastructure.html#the-singlecellexperiment-class) in particular helpful as you restructure this a tad @cbethell.